packages/fluxible-addons-react/.babelrc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "plugins": [
+        [
+            "@babel/plugin-transform-runtime",
+            {
+                "version": "7.14.0"
```

Comment:
Why is version specified?

Reply:
Replied here: https://github.com/yahoo/fluxible/pull/683#issuecomment-830857300